client/components/domains/use-your-domain-step/index.jsx

```diff
@@ -382,7 +382,7 @@ class UseYourDomainStep extends React.Component {
 			translate( "Requires changes to the domain's DNS" ),
 			this.getMappingPriceText(),
 		];
-		const buttonText = translate( 'Map Your Domain' );
+		const buttonText = translate( 'Map your domain' );
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.